### PR TITLE
feat(plugin): add loading animations

### PR DESCRIPTION
**Pull request by nina992**
# Overview 
Add loading animation in below places : 
- Datacatalog open button (カタログから検索する)
- Save field button (保存)
- Save template button (保存)
- Share button (共有)

### CODE REVIEW

File: plugin/web/extensions/sidebar/core/components/hooks.ts [hunk 1]
```diff
@@ -64,6 +64,8 @@ export default () => {
   const [fieldTemplates, setFieldTemplates] = useState<Template[]>([]);
   const [project, updateProject] = useState<Project>(defaultProject);
   const [selectedDatasets, setSelectedDatasets] = useState<DataCatalogItem[]>([]);
+  const [savingTemplate, setSaveTemplate] = useState<boolean>(false);
+  const [savingDataset, setSaveDataset] = useState<boolean>(false);
```
Comment (KaWaite) [inline plugin/web/extensions/sidebar/core/components/hooks.ts:68]: Is it possible just to have a single `loading` that is used for both template and dataset?

Comment (nina992) [reply plugin/web/extensions/sidebar/core/components/hooks.ts]: done

Comment (nina992) [reply plugin/web/extensions/sidebar/core/components/hooks.ts]: 8f47d1a450a11db460d7b822d7ed03fa51da839d

Comment (nina992) [reply plugin/web/extensions/sidebar/core/components/hooks.ts]: 2b0e25c711b0332eadd8ede255bf42f49778bfca

**Review by KaWaite — APPROVED** [review]

**Pull request merged**